src/main/java/com/gmail/nossr50/events/skills/taming/McMMOPlayerTameEntityEvent.java

```diff
@@ -0,0 +1,67 @@
+package com.gmail.nossr50.events.skills.taming;
+
+import com.gmail.nossr50.datatypes.experience.XPGainReason;
+import com.gmail.nossr50.datatypes.skills.PrimarySkillType;
+import com.gmail.nossr50.events.experience.McMMOPlayerExperienceEvent;
+import com.google.common.base.Preconditions;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Player;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Called when a player has tamed an entity, and we are about to award them experience for it.
+ */
+public class McMMOPlayerTameEntityEvent extends McMMOPlayerExperienceEvent {
+    private static final HandlerList HANDLER_LIST = new HandlerList();
+
+    private float xpGained;
+    private final Entity tamedEntity;
+
+    @ApiStatus.Internal
```

Comment:
Not only do we not make use of `ApiStatus.Internal` anywhere else, this being a public ctor is VERY much a public API... so my recommendation is you just remove this annotation